presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -316,4 +151,184 @@ public String toString()
                 .add("integerDictionaryEncodingEnabled", integerDictionaryEncodingEnabled)
                 .toString();
     }
+
```

Comment:
See if it makes sense to also add `OrcWriterOptions.toBuilder`.

Reply:
I will skip it for now.